core/apps/ame-e2e/src/integration/editor/edit-property.cy.ts

```diff
@@ -559,13 +552,9 @@ describe('Test edit property', () => {
         .then(() => cyHelp.clickSaveButton())
         .then(() => cy.getUpdatedRDF())
         .then(rdf => {
-          expect(rdf).to.contain(
-            ':Entity1 a samm:Entity;\n' +
-              '    samm:properties ([\n' +
-              '  samm:property :property2;\n' +
-              '  samm:notInPayload true\n' +
-              ']).',
-          );
+          expect(rdf).to.contain(':Entity1 a samm:Entity;\n');
+          expect(rdf).to.contain('samm:property :property2');
```

Comment:
Now we dont know if "samm:properties" contains the property2 and the notInPayload.